Services/Common/OJS.Services.Common.Models/Contests/Results/ContestResultsViewModel.cs

```diff
@@ -31,6 +31,10 @@ public class ContestResultsViewModel
 
     public bool UserHasContestRights { get; set; }
 
+    public int ItemsPerPage { get; set; }
+
+    public int PagesCount { get; set; }
```

Comment:
There is `PagedResults` property, why page info is needed here?

Reply:
Done.